sunpy/io/tests/test_ana.py

```diff
@@ -80,12 +78,13 @@ def test_f32c():
 
 @skip_ana
 def test_read_memmap():
-    # Test to check that passing memmap=True doesn't raise an error
+    # Test to check that passing memmap doesn't raise an error
     afilename = tempfile.NamedTemporaryFile().name
     ana.write(afilename, img_f32, 'testcase', 0)
-    img_f32c_rec = ana.read(afilename, memmap=True)
-    assert np.sum(img_f32c_rec[0][0] - img_f32) == 0
 
-    # Test to check that passing memmap=False doesn't raise an error
-    img_f32c_rec = ana.read(afilename, memmap=False)
-    assert not isinstance(img_f32c_rec, mmap.mmap)
+    # Memmap is not supported by ana
+    data, _ = ana.read(afilename, memmap=True)[0]
+    assert data.base is None
```

Comment:
Is it worth keeping the `assert np.sum(img_f32c_rec[0][0] - img_f32) == 0` check here?

Reply:
`ana.read` doesn't do anything with `**kwargs` so we aren't testing two different, yet equivalent, code pathways and verifying they give the same output. Adding a test to compare the two outputs would be part of a PR adding `mmap` support to `ana` in the future. Although maybe there isn't any harm keeping it in now to guard against breakages.

Reply:
I will add it back.